metasv/generate_sv_intervals.py

```diff
@@ -671,17 +664,21 @@ def generate_sc_intervals(bam, chromosome, workdir, min_avg_base_qual=SC_MIN_AVG
                                      min_soft_clip=min_soft_clip, max_nm=max_nm, min_matches=min_matches,
                                      skip_soft_clip=False, isize_mean=isize_mean, min_isize=min_isize, max_isize=max_isize)).sort().moveto(coverage_filtered_bed)
 
-        neigh_coverage_filtered_bed = os.path.join(workdir, "neigh_filtered_bp_merged.bed")
-        bedtool = bedtool.filter(lambda x: (float(x.fields[6]) * thr_sv[x.fields[3].split(",")[1]] <= float(x.fields[8]))).moveto(neigh_coverage_filtered_bed)
+        neigh_coverage_filtered_bed = os.path.join(workdir, "neigh_filtered.bed")
+        bedtool = bedtool.filter(lambda x: ((float(x.fields[6]) * thr_sv[x.fields[3].split(",")[1]] 
```

Comment:
Might be better to use a named function so that's it's easy to understand what filter we are using here.